blocks/api/post.pegjs

```diff
@@ -15,19 +22,35 @@ function maybeJSON( s ) {
 
 }
 
-Document
-  = WP_Block_List
+Block_List
+  = pre:$(!Token .)*
+    ts:(t:Token html:$((!Token .)*) { /** <?php return $t; ?> **/ return [ t, html ] })*
+    post:$(.*)
+  { /** <?php
+    $blocks = [];
+    if ( ! empty( $pre ) ) { $blocks[] = $pre; }
+    foreach ( $ts as $pair ) {
+      $blocks[] = $pair[ 0 ];
+      if ( ! empty( $pair[ 1 ] ) ) { $blocks[] = $pair[ 1 ] };
+    }
+    if ( ! empty( $post ) ) { $blocks[] = $post; }
 
-WP_Block_List
-  = WP_Block*
+    return $blocks;
+    ?> **/
 
-WP_Block
-  = WP_Tag_More
-  / WP_Block_Void
-  / WP_Block_Balanced
-  / WP_Block_Html
+    return [
+      freeform( pre ),
+      ...ts.reduce( ( out, [ t, h ] ) => [ ...out, t, freeform( h ) ], [] ),
```

Comment:
I recall we had issues previously because we don't run Babel on the `.pegjs` JavaScript, so I have a feeling this spread syntax won't work in IE11 and we'd either need to port to ES5 equivalent or investigate transpiling the result of the file.

Reply:
fixed in 4e41437d